formats/shared/definitions/_step_by_step_nav.jsonnet

```diff
@@ -0,0 +1,86 @@
+{
+  individual_step: {
+    type: "object",
+    additionalProperties: false,
+    required: [
+      "title",
+      "contents"
+    ],
+    properties: {
+      title: {
+        type: "string"
+      },
+      optional: {
+        type: "boolean"
+      },
+      contents: {
+        type: "array",
+        items: {
+          oneOf: [
+            { "$ref": "#/definitions/step_item_paragraph" },
+            { "$ref": "#/definitions/step_item_list" },
+          ]
+        }
+      }
+    }
+  },
+  step_item_paragraph: {
+    type: "object",
+    additionalProperties: false,
+    required: [
+      "type",
+      "text"
+    ],
+    properties: {
+      type: {
+        type: "string"
+      },
+      text: {
+        type: "string"
+      },
+    }
+  },
+  step_item_list: {
+    type: "object",
+    additionalProperties: false,
+    required: [
+      "type",
+      "contents"
```

Comment:
is style also a required field for this type?

Reply:
I think we good here, as for example `paragraphs` have no `style`

Reply:
This would only affect `step_item_list`